### PR TITLE
feat: changing way to detect crashlooping pods

### PR DESCRIPTION
**Pull request by benjaminch**

### CODE REVIEW

**Review by MacLikorne — APPROVED** [review]

**Review by deimosfr — DISMISSED** [review]

File: src/cmd/kubectl.rs [hunk 1]
```diff
@@ -1036,14 +1037,14 @@ pub fn kubectl_get_crash_looping_pods<P>(
...
+    let restarted_min = restarted_min_count.unwrap_or(10usize);
     let pods = kubectl_exec_get_pods(kubernetes_config, namespace, selector, envs)?;
 
     Ok(pods
         .items
         .into_iter()
         .filter(|pod| {
-            pod.status.phase != KubernetesPodStatusPhase::Running
+            pod.status.reason == KubernetesPodStatusReason::BackOff
                 && pod.status.container_statuses.as_ref().is_some()
```
Comment (deimosfr) [inline src/cmd/kubectl.rs:1048]: is it useful to check the status then? I'm not sure, if it's backofff, then...

Comment (benjaminch) [reply src/cmd/kubectl.rs:1048]: It is in case we want to kill if N restarts

Comment (deimosfr) [reply src/cmd/kubectl.rs]: May be you can add a comment, because it's not obvious

Comment (benjaminch) [reply src/cmd/kubectl.rs]: Added a comment, hpefuly it's clearer, let me know.

**Pull request merged**